test/test_integration_pr2.py

```diff
@@ -388,6 +385,29 @@ def test_UpdateGodMap(self, pocky_pose_setup):
         assert pocky_pose_setup.get_god_map().get_data(identifier.joint_cost + [u'odom_x_joint']) == 1
         assert pocky_pose_setup.get_god_map().get_data(identifier.joint_cost + [u'torso_lift_joint']) == 0.5
 
+    def test_UpdateYaml(self, pocky_pose_setup):
```

Comment:
Can you please add some tests, where the input is invalid and some where you try to modify other parts of the god map?